src/core/main.cpp

```diff
@@ -293,6 +297,13 @@ int main( int argc, char * * argv )
 	qInstallMessageHandler(consoleMessageHandler);
 #endif
 
+#if defined(LMMS_HAVE_SYS_PRCTL_H) && defined(PR_SET_CHILD_SUBREAPER)
+	// Set the "child subreaper" attribute so that plugin child processes remain as lmms'
+	// children even when some wrapper process exits, as it may happen with wine
+	if (prctl(PR_SET_CHILD_SUBREAPER, 1))
+		perror("prctl(PR_SET_CHILD_SUBREAPER)");
```

Comment:
Our coding conventions require explicit blocking for if-statements.
```suggestion
	if (prctl(PR_SET_CHILD_SUBREAPER, 1)) {
		perror("prctl(PR_SET_CHILD_SUBREAPER)");
	}
```

Reply:
Changed with an opening brace on its own line for consistency since all other code is like that in main.cpp.